p11-kit.yaml

```diff
@@ -85,12 +81,8 @@ subpackages:
     description: Server and client commands for p11-kit
     test:
       pipeline:
-        - uses: test/ldd-check
-          with:
-            packages: $(basename ${{targets.contextdir}})
-        - uses: test/ldd-check
-          with:
-            packages: p11-kit-server
+        - uses: test/tw/ldd-check
+        - uses: test/tw/ldd-check
 
```

Comment:
Once should be enough right? ;-)

Reply:
I didn't add the second one. but I've removed the dupes here now.